src/lev-report/ListFilter.js

```diff
@@ -0,0 +1,28 @@
+ import React from 'react';
+import styled from 'styled-components';
+import { InputField } from 'govuk-react';
+
+const StyledListFilter = styled(InputField)`
```

Comment:
StyledInputField?

Reply:
Now a data-list so changed to StyledDataList